Fluid/Filters/MiscFilters.cs

```diff
@@ -721,5 +721,25 @@ public static ValueTask<FluidValue> FormatString(FluidValue input, FilterArgumen
 
             return new StringValue(string.Format(culture, format, parameters));
         }
+
+        public static ValueTask<FluidValue> MD5(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            var value = input.ToStringValue();
+            if (String.IsNullOrEmpty(value))
+            {
+                return StringValue.Empty;
+            }
+
+            using (var provider = System.Security.Cryptography.MD5.Create())
+            {
+                var builder = new StringBuilder();
```

Comment:
Initialize size with 32 and there will be only a single array allocation.
Or use `String.Create` and then it's one less allocation better.

Reply:
There's also StringBuilderPool  😉